### DISCUSSION TITLE
My Solution to fixing the Scrolling Problem.

### DISCUSSION THREAD

**Discussion by NX721** [Prodigy-Hacking/ProdigyMathGameHacking#714] [(C) Suggestions]
So, many of you players are complaining about how you can't scroll down with the menu.
My solution to this is by pressing the 2 buttons: "Ctrl" and the "-" (Minus) button. This will shrink the cheat menu allowing you to use the button "Enable Resize Drag" . Then resize it to a fitting size, press "Ctrl" and "+" to return everything to the original size and you can now scrolling is enabled.

**Comment by PatheticMustan** [comment]
temporary fix, but somebody needs to unfuck the css

**Reply by NX721** [reply]
True.
